tests/suite/system_suite_test.go

```diff
@@ -69,9 +95,18 @@ var _ = BeforeSuite(func() {
 
 	output, err = framework.InstallNGF(cfg)
```

Comment:
do we need to start using/passing timeouts in NGF installation/removal methods? Similarly to how we use them in Gateway resources and test manifests?

Reply:
I think we can cross that bridge when we come to it. If helm starts timing out, then we can make the timeout configurable. For now, it'll use the default helm timeout.